fetch.js

```diff
@@ -0,0 +1,32 @@
+import {
+    SERVER_URL_WEATHER,
+    SERVER_URL_FORECAST,
+    API_KEY,
+} from "./constants.js";
+
+export const getWeatherUrl = function (value) {
+    return `${SERVER_URL_WEATHER}?q=${value}&appid=${API_KEY}`;
+}
+
+export const getForecastUrl = function (value) {
+    return `${SERVER_URL_FORECAST}?q=${value}&appid=${API_KEY}`;
+}
+
+export const fetchWeatherData = function (value, url) {
+    return fetch(url(value))
```

Comment:
вот тут перемудрил, сложно читается fetch
лучше сделать так:
```suggestion
    export const fetchWeatherData = (serverUrl, cityName) => {
    const url = getUrl(serverUrl, cityName);
    return fetch(url)
    ...
```